<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWidget</class>
 <widget class="QWidget" name="SelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="returnButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>主菜单</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>学号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>120</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>姓名</string>
   </property>
  </widget>
  <widget class="QLabel" name="showNameLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QLabel" name="mathLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>72</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>数学成绩</string>
   </property>
  </widget>
  <widget class="QLabel" name="showMathLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>170</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QLabel" name="enLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>220</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>英语成绩</string>
   </property>
  </widget>
  <widget class="QLabel" name="showEnLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>215</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QLabel" name="cLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>270</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>C语言成绩</string>
   </property>
  </widget>
  <widget class="QLabel" name="showCLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>265</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QLabel" name="avgLabel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>320</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>平均成绩</string>
   </property>
  </widget>
  <widget class="QLabel" name="showAvgLabel">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>315</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>NULL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectButton">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>查询</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
